src/utils.js

```diff
@@ -0,0 +1,15 @@
+export const checkForAlpine = () => {
```

Comment:
maybe not for this PR, but there could also be a check that "Alpine version > version that added `addMagicProperty`"

Reply:
https://github.com/alpine-collective/alpine-magic-helpers/issues/15